src/features/blocks/Block.tsx

```diff
@@ -1,51 +1,80 @@
 import './Block.css'
 import {ToolboxCategoryBlocks} from "../../toolboxconfig";
 import {useEffect, useRef} from "react";
-import {useAppDispatch} from "../../hooks";
+import {useAppDispatch, useAppSelector} from "../../hooks";
 import {toggleBlockDescriptionAction} from "../../store/features/blocks/blocks";
+import {Draggable} from "react-beautiful-dnd";
+import {getDragStyle} from "../../App";
 
 function Block(props: ToolboxCategoryBlocks) {
+  const cat = useAppSelector(state => state.blocksReducer.categories.find((c) => c.blocks.find((b) => b.id === props.id)))
```

Comment:
You don't need the parenthesis when there is only one argument to the lambda function, eg:

`elements.find(c => condition(c))`